h/api_client/subscribers.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import pyramid.events
+
+import h.api_client.api_client as api_client
+
+
+def _get_api_url(request):
+    """Return the configured base URL for this app's Hypothesis API instance.
+
+    Returns the value of the h.api_url setting from the config file,
+    or if that doesn't exist the "/api" route from the WSGI app (which assumes
+    that we're running the app with the "api" feature enabled).
+
+    Always returns the URL _without_ a trailing /, despite what the user might
+    have entered in the config file.
+
+    """
+    return request.registry.settings.get(
+        "h.api_url", request.resource_url(request.root, "api")).rstrip("/")
```

Comment:
Chatted with some people in #pyramid about this (they're really helpful). I think the best we can do is:

1. Remove the trailing / at boot time in includeme() like tilgovi says
2. Add a method to the request object to return the api url, defaulting to the route computed from the request object (this can't be done at boot time b/c there's no request yet)